packages/generator/templates/helpers.ts.hbs

```diff
@@ -2,31 +2,67 @@
 import { ShopperToken, stripBearer } from "@commerce-apps/core"
 import  * as sdk from "./"
 
-
-const shopperAuthApi = sdk.{{shopperAuthClient}}.Client.prototype.{{shopperAuthApi}};
+//type of the input parameter of auth function
+type authFuncParamType = Parameters<typeof sdk.{{shopperAuthClient}}.Client.prototype.{{shopperAuthApi}}>[0];
 
 /**
  * @description This wraps the parameters for the authCall to retrieve a token. 
```

Comment:
might want to detail what we mean by "authCall". Perhaps have an example of what the call should look like